visualization/app/codeCharta/util/schema.json

```diff
@@ -20,7 +20,8 @@
 		},
 		"apiVersion": {
 			"title": "version of schema",
-			"type": "string"
+			"type": "string",
+			"pattern": "[0-9][.][0-9]"
```

Comment:
I am not a regex expert, but isn't backslashing a dot (like `"[0-9]\.[0-9]"`) the more often used option of matching a dot?

Reply:
We didn't change our apiVersion often, but with this pattern a `1.10` would be be considered invalid

Reply:
This was done another way now, I forgot to remove those changes, thank you

Reply:
([0-9]+)\\.([0-9]+) should be the required solution

Reply:
I think you can remove the round brackets and one backslash 😉

Reply:
double slash is needed or parsing will fail, wrong espace sequence

Reply:
We need to use \\ to escape the escape character as the whole thing will be converted to string later, should be fixed now